server/package.json

```diff
@@ -4,6 +4,14 @@
   "description": "Jax Pet Names API",
   "main": "index.js",
   "license": "MIT",
+  "scripts": {
+    "db:create": "node knex_create.js",
+    "db:drop": "node knex_drop.js",
+    "db:migrate": "knex migrate:latest",
```

Comment:
I finally found the knex command for running _all_ migrations!

Reply:
excellent!